### DISCUSSION TITLE
Help needed

### DISCUSSION THREAD

**Discussion by khagenbaro** [hub4j/github-api#1721] [General]
I am new to this project . I want to contribute . 
Let me introduce myself first I am khagen baro from india . I have around 2 years of experience . I have worked with java spring boot , spring data jpa , hibernate , little bit spring security . 
Can anyone quickly help me for initial set up and running the application in my local.
I have forked the repo

**Comment by bitwiseman** [comment]
@khagenbaro
See https://github.com/hub4j/github-api/blob/main/CONTRIBUTING.md  .  If that doesn't have sufficient instructions, please submit PR to update it.
